tests/dopamine/replay_memory/circular_replay_buffer_test.py

```diff
@@ -680,6 +680,38 @@ def testMemoryHasASpecifiedLock(self):
     memory = _create_dummy_memory(lock='lock-name')
     self.assertEqual(memory._lock, 'lock-name')
 
+  def testAddNodeToTrajectoryBuffer(self):
+    memory = circular_replay_buffer.OutOfGraphReplayBuffer(
+        observation_shape=OBSERVATION_SHAPE,
+        stack_size=STACK_SIZE,
+        replay_capacity=5,
+        batch_size=BATCH_SIZE,
+        contiguous_trajectories=True)
+    self.assertEqual(memory.cursor(), 0)
+    self.assertEqual(len(memory._trajectories), 0)
+    zeros = np.zeros(OBSERVATION_SHAPE)
+    memory.add(zeros, 0, 0, 0)
+    self.assertEqual(memory.cursor(), 0)
```

Comment:
I think if I was writing these, I'd break it up into two different tests: one checking that the step isn't added to main replay buffer and one checking that it is added to a trajectory buffer.  This preference comes from "test behaviors not functions."  I also think it's fine as written.

Reply:
done. split in two tests.